src/daft-micropartition/src/python.rs

```diff
@@ -28,6 +28,18 @@ struct PyMicroPartition {
     inner: Arc<MicroPartition>,
 }
 
+// TODO remove this repeated code fragment
```

Comment:
I think we could get rid of this duplicated code by changing `daft_csv::read::read_csv()` from taking `Option<u8>` for `delimiter`, `quote`, `escape_char`, and `comment`, to taking `Option<char>`. Then we could have a single `char_to_byte` utility that lives in `daft_csv` and that's applied to each of these arguments as they come in to `read_csv`.